### PR TITLE
CRM: Resolves 3119 - handle WP update pages in tests

### PR DESCRIPTION
**Pull request by tbradsha**
Resolves Automattic/zero-bs-crm#3119 - handle WP update pages in tests

## Proposed changes:

Our Codeception tests are fairly fragile. When WP has a database update it loads a page for the end user which our tests couldn't handle.

This PR adds a test for the "update database" page as well as the "admin email verification" page. I've also updated the database to the latest version and reenabled tests that were temporarily disabled in #30917.

Reported in p1684954743887389-slack-C034JEXD1RD; triggered by this commit:
https://github.com/WordPress/wordpress-develop/commit/52f05444bab8659076a3cd6b5282ea44af44da86

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

Make sure tests on this PR successfully ran against WP `trunk`.

### CODE REVIEW

**Comment by tbradsha** [comment]
[Appears to have been successful](https://github.com/Automattic/jetpack/actions/runs/5074031446/jobs/9113775431?pr=30925).

**Review by gogdzl — APPROVED** [review]
LGTM!

File: projects/plugins/crm/tests/action-skip-test-php.sh [hunk 1]
```diff
@@ -5,8 +5,8 @@ if php -r 'exit( version_compare( PHP_VERSION, "7.2.0", "<" ) ? 0 : 1 );'; then
...
-# TODO Fix the bug and remove this
-if [[ "$WP_BRANCH" == 'trunk' ]]; then
-    echo "Temporarily disabling tests against WP trunk until we get around to updating our codeception DB image."
-    exit 3
-fi
+# Uncomment the below snippet to disable tests on WP trunk
+# if [[ "$WP_BRANCH" == 'trunk' ]]; then
+# 	echo "Codeception tests against WP trunk are temporarily disabled."
+# 	exit 3
+# fi
```
Comment (kallehauge) [inline projects/plugins/crm/tests/action-skip-test-php.sh:12]: Can we remove this as well? 😄

Comment (tbradsha) [reply projects/plugins/crm/tests/action-skip-test-php.sh:12]: I initially was planning to remove it (I typically don't like commented + unused code), but figured I'd leave it in our tooling as an example of a quick way to disable tests should something similar come up.

**Pull request merged**